website/pages/docs/post-processors/googlecompute-export.mdx

```diff
@@ -28,47 +25,18 @@ credentials.
 As such, the authentication credentials that built the image must have write
 permissions to the GCS `paths`.
 
+~> **Note**: By default the GCE image being exported will be deleted once the image has been exported.
+To prevent Packer from deleting the image, the `keep_input_artifact` can be set to `true`. See [Input Artifacts](https://www.packer.io/docs/templates/post-processors#input-artifacts) for more details.
```

Comment:
```suggestion
To prevent Packer from deleting the image set the `keep_input_artifact` configuration option to `true`. See [Post-Processor Input Artifacts](https://www.packer.io/docs/templates/post-processors#input-artifacts) for more details.
```